cub/examples/device/example_device_topk_keys.cu

```diff
@@ -42,19 +45,22 @@ void Initialize(float* h_keys, float* h_reference_keys, int num_items, int k)
 
   if (g_verbose)
   {
-    printf("Input keys:\n");
+    std::cout << "Input keys:\n";
```

Comment:
Remark: I love how "modernizing" means to go from `printf` to `cout` :) But `std::print` is probably a bit too far out yet.